application/F3DOptionsParser.cxx

```diff
@@ -101,7 +102,7 @@ class ConfigurationOptions
   {
     // cxxopts do not support an empty string as a value
     // for vector<double> and vector<int>
-    assert(currValue.size() > 0);
+    // assert(currValue.size() > 0);
```

Comment:
but it supports it for a std::vector<std::string>, so removing the assert makes complete sense. We just need to make sure we do not empty initialize any option.